docs/troubleshooting/supported-browsers.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Supported Browsers List
+image: "/docs/images/docs-meta-cards/troubleshoot-card.png"
+displayed_sidebar: docs
+description: "Webpack 5 Issues | Documentation - Web3Auth"
+---
+
+---
+
+## Supported Browsers List
+
+Web3Auth SDKs have functions which require BigInt compatability and thus the browsers which are supported are as follows. The browsers are listed
```

Comment:
In this sentence, both the lines convey the same thing, can we merge those together and construct one line?

Reply:
I agree, please remove the H2 and add description properly on why and how this error is happening. Add the link: https://browsersl.ist/#q=supports+bigint%2C+not+dead